### PR TITLE
Pull releases from github

### PR DESCRIPTION
**Pull request by craigmulligan**
Connects to #81 
Also added prettier, to keep things tidy. (will format pages/*) in a seperate PR.
We also probably need a github token now for the CI seen as we are pulling from githubs api again. 

Change-Type: patch

### CODE REVIEW

File: lib/getLatestRelease.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import includes from 'lodash/includes';
+import async from 'async';
+import GitHubApi from 'github';
+
+var github = new GitHubApi();
```
Comment (Shou) [inline lib/getLatestRelease.js]: Should we use `const` here instead?

File: lib/getLatestRelease.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    )}`;
+  } else {
+    return `Etcher for ${getOS(file)} ${getArch(file)} ${getArchString(
+      file
+    )} ${getType(file)}`;
+  }
+};
+
+const prepAssets = assets => {
+  // console.log(assets)
```
Comment (Shou) [inline lib/getLatestRelease.js]: I assume this can be removed now

File: lib/fetchData.js [hunk 3]
```diff
@@ -9,34 +9,46 @@ let data;
...
-    fetch('https://raw.githubusercontent.com/resin-io/etcher/master/docs/CLI-INSTALLATION.md')
-    .then(res => ( res.text() )),
-    fetch('https://raw.githubusercontent.com/resin-io/etcher/master/CHANGELOG.md')
-    .then(res => ( res.text() )),
-    S3(locals.s3Bucket, locals.title.toLowerCase())
+    fetch(
+      'https://raw.githubusercontent.com/resin-io/etcher/master/docs/CLI-INSTALLATION.md'
+    ).then(res => res.text()),
+    fetch(
+      'https://raw.githubusercontent.com/resin-io/etcher/master/CHANGELOG.md'
```
Comment (Shou) [inline lib/fetchData.js]: Maybe we can store the URLs as constants? I imagine this should make these lines easier to read, e.g. `fetch(ETCHER_RAW_CHANGELOG_URL)` or something like that.

**Review by Shou — COMMENTED** [review]
Works well! Other than a few minor things, this looks pretty great.

File: lib/getLatestRelease.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+import GitHubApi from 'github';
+
+var github = new GitHubApi();
+
+if (process.env.GH_TOKEN) {
+  github.authenticate({
+    type: 'token',
+    token: process.env.GH_TOKEN
+  });
+}
```
Comment (lurch) [inline lib/getLatestRelease.js:12]: Is there / does there need to be any error-handling for if `GH_TOKEN` either isn't set, or is invalid?

Comment (craigmulligan) [reply lib/getLatestRelease.js:12]: No it only needs to be set if we get rate limited by github

**Pull request merged**